frontends/web/src/locales/es/app.json

```diff
@@ -1068,8 +1068,8 @@
     "title": "Nota"
   },
   "notification": {
-    "newTxs": "Nueva transacción en: {{accountName}}",
-    "newTxs_plural": "{{count}} nuevas transacciones en: {{accountName}}"
+    "newTxs_one": "Nueva transacción en: {{accountName}}",
```

Comment:
tbh i don't know what happens if you locally change stuff in languages other than `en` and then do a locize push. i guess we will see... so far we have only made local additions/deletions/renames in `en` and then pushed.

Reply:
I didn't know that, and Locize did not loose and also rename the keys in the other languages?

Reply:
This PR also needs this PRhttps://github.com/digitalbitbox/bitbox-wallet-app/pull/1550